v_4_9_1/release.go

```diff
@@ -0,0 +1,62 @@
+package v_4_9_1
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/release/v1alpha1"
+)
+
+func BuildImages(registryDomain string, versions Versions) Images {
```

Comment:
It's not totally clear where this code should live. This seemed like as good a place as any.